src/app/components/filters/stops-filter/stops-filters.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { StopsFilterProps, StopsFilter } from "./stops-filter";
+import { Stops } from "../../../models/flight-parameters";
+
+// TODO discuss how this could be made better
+// right now the mock recreates what external component might do
+jest.mock('../../../../components/ui/button', () => ({
+  Button: ({children, ...props}: any) => {
+    return <button {...props}>{ ...children }</button>;
+  }
+}));
+
+describe(StopsFilter.name, () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should throw an error if setActiveFilter property is not sent', () => {
+    const props: StopsFilterProps = {
+      setActiveFilter: undefined as any
+    };
+    
+    expect(() => render(<StopsFilter {...props}></StopsFilter>))
```

Comment:
This rendering can go inside beforeEach block, by that way there is no need to repeat inside every test
.